tests/test_pkcs1.py

```diff
@@ -106,3 +106,35 @@ def test_multiple_signings(self):
         signature2 = pkcs1.sign(message, self.priv, 'SHA-1')
 
         self.assertEqual(signature1, signature2)
+
+    def test_split_hash_sign(self):
+        """Hashing and then signing should match with directly signing the message. """
+
+        message = b'je moeder'
+        print("\tMessage:   %r" % message)
```

Comment:
I was using the other test cases in this file as a template and many of them had prints.  The prints are now removed.